templates/actor/shared/effects.hbs

```diff
@@ -1,63 +1,79 @@
 {{! Effects Tab }}
-<section class="tab effects flexcol {{tab.cssClass}}" data-group="primary" data-tab="effects">
-  <ol class="effects-list">
-    {{#each effects as |section sid|}}
-    <li class="effects-header flexrow draggable" data-effect-type="{{section.type}}">
-      <div class="effect-name flexrow">
-        {{localize section.label}}
-      </div>
-      <div class="effect-source">
-        {{localize "DRAW_STEEL.Effect.Source"}}
-      </div>
-      <div class="effect-source">{{localize "EFFECT.TABS.duration"}}</div>
-      <div class="effect-controls flexrow">
-        <a
-          class="effect-control"
-          data-action="createDoc"
-          data-document-class="ActiveEffect"
-          data-origin="{{@root.document.uuid}}"
-          data-img="icons/svg/aura.svg"
-          {{#if (eq section.type "inactive")}}
-          data-disabled="true"
-          {{! eslint-disable-next-line @html-eslint/no-duplicate-attrs }}
-          {{else if (eq section.type "temporary")}}
-          data-duration.rounds="1"
-          {{/if}}
-          data-tooltip="{{localize "DOCUMENT.Create" type=(localize "DRAW_STEEL.Effect.Name")}}"
-          >
-          <i class="fa-solid fa-plus"></i>
-          {{localize "DOCUMENT.New" type=(localize "DRAW_STEEL.Effect.Name")}}
-        </a>
-      </div>
-    </li>
-
+<section class="tab effects {{tab.cssClass}} standard-form" data-group="primary" data-tab="effects">
+  <fieldset class="status-effects">
+    <legend>{{localize "DRAW_STEEL.Effect.StatusConditions"}}</legend>
+    <div class="status-container">
+      {{#each statuses as |status id|}}
+      <button type="button" class="status-control {{status.active}}" {{status.disabled}} data-status-id="{{id}}" data-tooltip-html="{{status.tooltip}}" data-action="toggleStatus">
+        <img src="{{status.img}}" alt="{{status.name}}">
+        <span>{{status.name}}</span>
```

Comment:
I think it would be clearer what is happening if we used the `{{disabled}}` helper.